dtfs-central-api/src/v1/routes/tfm-routes.js

```diff
@@ -507,52 +507,6 @@ tfmRouter.route('/facilities/:facilityId/amendments').post(tfmPostAmendmentContr
  */
 tfmRouter.route('/facilities/:facilityId/amendments/:amendmentId').put(tfmPutAmendmentController.updateTfmAmendment);
 
-/**
```

Comment:
Any reason these endpoints are no longer required?

Reply:
Because they were only used for mock data loader and are never actually used in a service